moveos/moveos-stdlib/src/natives/mod.rs

```diff
@@ -1,25 +1,25 @@
 // Copyright (c) RoochNetwork
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::addresses::{MOS_STD_ADDRESS, MOVE_STD_ADDRESS};
+use crate::addresses::{MOVEOS_STD_ADDRESS, MOVE_STD_ADDRESS};
 use move_core_types::gas_algebra::InternalGas;
 use move_vm_runtime::native_functions::{make_table_from_iter, NativeFunctionTable};
 
 mod helpers;
-pub mod mos_framework;
-pub mod mos_stdlib;
+pub mod moveos_stdlib;
+pub mod rooch_framework;
 
 #[derive(Debug, Clone)]
 pub struct GasParameters {
     move_stdlib: move_stdlib::natives::GasParameters,
     move_nursery: move_stdlib::natives::NurseryGasParameters,
     table_extension: move_table_extension::GasParameters,
-    type_info: mos_stdlib::type_info::GasParameters,
-    rlp: mos_stdlib::rlp::GasParameters,
-    account: mos_framework::account::GasParameters,
-    bcd: mos_stdlib::bcd::GasParameters,
-    tx_context: mos_stdlib::tx_context::GasParameters,
-    object: mos_stdlib::object::GasParameters,
+    type_info: moveos_stdlib::type_info::GasParameters,
+    rlp: moveos_stdlib::rlp::GasParameters,
+    account: rooch_framework::account::GasParameters,
```

Comment:
If moveos split to an independent library，the `GasParameters` in moveos can‘t depend rooch, do we need to move `GasParamerters` into rooch_framework?